pkg/controller/machine.go

```diff
@@ -424,7 +426,11 @@ func (c *Controller) updateMachineStatus(machine *machinev1alpha1.Machine, node
 }
 
 func parseContainerRuntime(s string) (runtime, version string, err error) {
-	re, _ := regexp.Compile(`(\w*)://(.*)`)
+	regex := `(\w*)://(.*)`
+	re, err := regexp.Compile(regex)
```

Comment:
This is one of the rare cases where I actually put things into globals. This way they won't be compiled every time the func is called.

Reply:
yeah and in this case you can use `regexp.MustCompile` and assign the result in an var-block